EIPS/eip-5505.md

```diff
@@ -0,0 +1,83 @@
+---
+eip: 5505
+title: EIP-1155 asset backed NFT extension
+description: Extends EIP-1155 to support crucial operations for asset backed NFTs.
+author: liszechung (@liszechung)
+discussions-to: https://ethereum-magicians.org/t/eip-draft-erc1155-asset-backed-nft-extension/10437
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-18
+requires: 1155
+---
+
+## Abstract
+To propose an extension of smart contract interfaces for asset-backed, fractionalized projects using EIP-1155 standard such that total acquisition will become possible. This proposal focuses on In-Real-Life asset, where total acquisition should be able to happen.
```

Comment:
```suggestion
To propose an extension of smart contract interfaces for asset-backed, fractionalized projects using the [EIP-1155](./eip-1155.md) standard such that total acquisition will become possible. This proposal focuses on physical assets, where total acquisition should be able to happen.
```
Required: I don't understand the first sentence; it appears to be incomplete. Please fix.
Required: EIP-1155 needs to be linked
Required: Use the correct terminology